src/esp/gfx/MaterialUtil.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "MaterialUtil.h"
+
+#include <Magnum/Trade/PbrMetallicRoughnessMaterialData.h>
+#include <Magnum/Trade/PhongMaterialData.h>
+
+namespace Mn = Magnum;
+
+namespace esp {
+namespace gfx {
+
+gfx::PhongMaterialData::uptr buildPhongFromPbrMetallicRoughness(
```

Comment:
This function is a temporary hack to approximate a PBR material with Phong material properties. See screenshots in the PR description. This function should go away in a few months once we have a PBR shader.